javascript_nodejs/4.simple-prompt-bot/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "console-echobot",
```

Comment:
this neeeds to be simple-prompt-bot or something to that effect. Most certainly not console-echobot :)